Documentation/Login/Index.rst

```diff
@@ -0,0 +1,25 @@
+.. include:: ../Includes.txt
+
+.. _login:
+
+=====
+Login
+=====
+
+In TYPO3 CMS an editor's work is done in the backend. 
+
+.. important::
+
+   Make sure JavaScript and cookies are enabled in your browser as they are
+   required by the TYPO3 Backend.
+
+Enter your domain name into the address bar of your internet browser and
+append "/typo3" to it to get to the "log in" page of your site.
+For example: :samp:`http://www.example.org/typo3`.
+
+.. figure:: ../Images/BackendLoginPage.png
```

Comment:
Sorry, just found now: The image path should be `/Images/AutomaticScreenshots/Login/BackendLoginPage.png`